subworkflows/nf-core/fastq_extract_kraken_krakentools/main.nf

```diff
@@ -0,0 +1,24 @@
+include { KRAKEN2_KRAKEN2                } from '../../../modules/nf-core/kraken2/kraken2/main'
+include { KRAKENTOOLS_EXTRACTKRAKENREADS } from '../../../modules/nf-core/krakentools/extractkrakenreads/main'
+
+workflow FASTQ_EXTRACT_KRAKEN_KRAKENTOOLS {
+
+    take:
+    ch_reads  // channel: [ val(meta), path(reads) ]
+    ch_db     // channel: [ path db ]
+    val_taxid // string: taxinomic ids, separated by spaces
+
+    main:
+    ch_versions = Channel.empty()
+
+    KRAKEN2_KRAKEN2 ( ch_reads, ch_db, true, true )
+    ch_versions = ch_versions.mix(KRAKEN2_KRAKEN2.out.versions.first())
```

Comment:
Nothing to be exported to MultiQC?

Reply:
I just checked - only the Kraken2 reports would be relevant. The subworkflow emits the Kraken2 reports as `kraken2_report`. 

Should there always be an output `multiqc` or `multiqc_files` from subworkflows? I couldn't find something in the guidelines, but it would totally make sense.

Reply:
Yes I think it would make sense, but maybe we can ask @maintainers-team ? (If that works ok GitHub...)

Reply:
Ask on slack: yes, when compatible output file with MQC :)

Reply:
Is there a convention for the name of the channel? My random search gave me `multiqc` and `multiqc_files` 
(I couldn't find it on Slack - feel free to mention me there, if you want!)

Reply:
In pipeline code I usually use `ch_multiqc_files` so emission as `multiqc_files` is probably fine 👍

The chat was in the maintainers team channel which is probably why! Wasn't much discussion, just yes we should 😬